pkg/compiler/debug_test.go

```diff
@@ -42,6 +42,8 @@ func MethodByteArray() []byte { return nil }
 func MethodArray() []bool { return nil }
 func MethodStruct() struct{} { return struct{}{} }
 func unexportedMethod() int { return 1 }
+type MyStruct struct {}
+func (ms MyStruct) MethodOnStruct() { }
```

Comment:
What about `func (ms *MyStruct) MethodOnPointerToStruct() { }`?

Reply:
Add one more test and found a bug.

Reply:
Yeah, that's what happens when you add tests!